From e2dff24c2d99e91c88a5532405a668f995b24a86 Mon Sep 17 00:00:00 2001
From: Jackie Huang <jackie.huang@windriver.com>
Date: Mon, 27 Apr 2020 11:55:33 +0800
Subject: [PATCH 4/4] configure.ac: fix install-sh not found

Fix configure.ac to cope with new autoconf.
Recent autoconfs generate a bad configure when AM_INIT_AUTOMAKE is
called as late as it was, ending up thinking that the am_aux_dir is pwd
at the start of the build. Move it up to under AC_INIT to fix that.

Author: William Grant <wgrant@ubuntu.com>

Upstream-Status: Pending

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=724116

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for 4.1.1

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
---
 configure.ac | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4d5fb96..af7f646 100644
--- a/configure.ac
+++ b/configure.ac
@@ -23,13 +23,20 @@ AC_INIT([resource-agents],
 	m4_esyscmd([make/git-version-gen .tarball-version]),
 	[to_be_defined@foobar.org])
 
+AC_CONFIG_AUX_DIR(.)
+
+dnl
+dnl AM_INIT_AUTOMAKE([1.11.1 foreign dist-bzip2 dist-xz])
+dnl
+
+AM_INIT_AUTOMAKE([1.10.1 foreign dist-bzip2])
+
 AC_USE_SYSTEM_EXTENSIONS
 
 CRM_DTD_VERSION="1.0"
 
 PKG_FEATURES=""
 
-AC_CONFIG_AUX_DIR(.)
 AC_CANONICAL_HOST
 
 dnl Where #defines go (e.g. `AC_CHECK_HEADERS' below)
@@ -84,12 +91,6 @@ AC_ARG_WITH([systemdtmpfilesdir],
      fi
 AM_CONDITIONAL(HAVE_SYSTEMD, [test -n "$with_systemdtmpfilesdir" -a "x$with_systemdtmpfilesdir" != xno ])
 
-dnl 
-dnl AM_INIT_AUTOMAKE([1.11.1 foreign dist-bzip2 dist-xz])
-dnl
-
-AM_INIT_AUTOMAKE([1.10.1 foreign dist-bzip2])
-
 AC_DEFINE_UNQUOTED(AGENTS_VERSION, "$PACKAGE_VERSION", Current agents version)
 
 CC_IN_CONFIGURE=yes
-- 
2.7.4

